src/app/lib/streamer.js

```diff
@@ -324,7 +271,7 @@
                             info: torrent,
                             subtitle: model.get('subtitle'),
                             defaultSubtitle: model.get('defaultSubtitle'),
-                            title: title,
+                            title: model.get('title'),
```

Comment:
I haven't noticed anything else, no. It's good work, that's sure. I just point out what I see^^